scripts/autogen.py

```diff
@@ -392,6 +394,12 @@ def sync_tutobook_templates(self):
                 os.makedirs(dst_dir)
                 copy_inner_contents(dir_path / "md", dst_dir, ext=".md")
 
+        shutil.copytree(
```

Comment:
Add a TODO here?

Reply:
No, this isn't a TODO. We have to copy over the examples to KerasRS